tox.ini

```diff
@@ -47,31 +48,31 @@ deps =
     twython
 
 commands =
-    pytest
```

Comment:
Would it be possible to remove this line completely? If I understood tox correctly, `commands` should be inherited from `testenv`. As you have yourself experienced, the reduction in duplication should make future maintenance easier.

Reply:
good eye! since pypy comes bundled with numpy, we can keep the parent `commands`. but with the nodeps environments, we need to redefine `commands` because otherwise it would try to install scipy and scikit-learn to those environments.